### ISSUE TITLE
Smart Menus: Improve the UX and accessibility of the FontAwesome chooser

### ISSUE THREAD

**Issue by abias** [moodle-an-hochschulen/moodle-theme_boost_union#452]
The FontAwesome chooser which has been added to the menu item settings is a nice tool to quickly select an icon for a menu item.

However, I think the UX can be improved. I see at least these two downsides:
* The base of the widget is just a text field. The user might not understand that he has to click into the field to show the chooser. Something like a hint saying "Click to select icon" which is shown in the text field as long as no icon is selected might be useful.
* The FontAwesome chooser is not activated if you tab through the menu items form. There should be a possibility to use the chooser with the keyboard only for accessibility reasons as well.
